experiments/dataSources/airtable/README.md

```diff
@@ -81,20 +79,98 @@ If you've set the data source URL correctly, clicking on **Fields** will show yo
 
 You'll want to select Id, Created Time (if you prefer), and then you'll want to click into the Fields "field." You'll find your sheet's columns listed here. Select all relevant fields, and then build your page!
 
+
+## Lets make this DataSource Full CRUD! 
+
+Reading data is nice.  But when you can read and write.. Why not!!   Skuid supports all the HTTP verbs - so you can Create,  Update and Delete  (as well as just Read). 
+
+Go back to your Read Only model and make the following change: 
+
+- **Model Behavior**: Read/Write
+
+This exposes a "Methods" area of the model,  and moves your Query action there. Now we'll add separate entries for the other methods for doing full CRUD.   Using the Airtable API docs we can configure each of these methods. 
+
+### Delete
+Configuring delete is easiest. 
+
+- **Type**: "DELETE"
+- **Data Source URL**: ``<baseId>/<ObjectName>/{{id}}``
+- **Data Source HTTP Verb**: ``DELETE``
+
+This Data Source URL takes the ID of one record and passes it to Airtable to delete.  You can use the "Mark row(s) for deletion" and "Save" actions to trigger this method.  
+
+### Insert
+Lets create a new record. 
+
+- **Type**: "INSERT"
+- **Data Source URL**: ``<baseId>/<ObjectName>``
+- **Data Source HTTP Verb**: ``POST``
+- **Send new field values**: ``As templated request body``
+- **Method succeeds if...**: ``Request Succeeds``
+- **Response handling**: ``Created Record will be returned``
+
+You configure exactly how the new fields are passed to Airtable in the Custom Request template. 
+Below is the template for creating a record with all the fields in the applicants table. 
+
+```json
+{"fields": {
```

Comment:
The snippet would only be a few lines long. That's a far cry from a new data source and would go a long way to making this more maintainable.

Just want to point out that there are very appropriate places to use snippets, and this IMO is one of them. A litmus test could be that if a small bit of code can be written that enables more productive declarative development, and encourages additional declarative development rather than discouraging it, that piece of code is worth writing.

Reply:
Ok.  I'm open to another PR.

Reply:
But maybe we leave both methods up as documentation of the merge template concept.